### PR TITLE
Add org.bzflag.BZFlag

### PR DESCRIPTION
**Pull request by rrigby**
https://github.com/BZFlag-Dev/bzflag

### CODE REVIEW

**Comment by barthalion** [comment]
Please contact upstream developers if they would like to (co-)maintain it on Flathub or have write access just in case and contribute appdata there as well.

bot, build org.bzflag.BZFlag

File: org.bzflag.BZFlag.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        },
+        {
+          "type": "file",
+          "path": "org.bzflag.BZFlag.appdata.xml"
+        }
+      ],
+      "post-install": [
+        "install -Dm644 data/bzflag.desktop /app/share/applications/bzflag.desktop",
+        "install -Dm644 data/red_icon.png /app/share/icons/hicolor/64x64/apps/bzflag.png",
+        "install -Dm644 org.bzflag.BZFlag.appdata.xml /app/share/appdata/org.bzflag.BZFlag.appdata.xml"
```
Comment (eszlari) [inline org.bzflag.BZFlag.json]: `"install -Dm644 -t /app/share/metainfo org.bzflag.BZFlag.appdata.xml"`

**Comment by allejo** [comment]
Hi there! Since none of the project leads have joined this conversation, I'll hop in on the project's behalf as a developer. Exploring Flatpak is something we had definitely been wanting to do, so thanks for the push of getting this started.

Flatpak is new to our team but we'd be glad to learn and help maintain it. A few questions,

- Are there any specific resources we should familiarize ourselves with for becoming maintainers?
- What does having write access entail?

File: org.bzflag.BZFlag.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+  "id": "org.bzflag.BZFlag",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "bzfs",
```
Comment (allejo) [inline org.bzflag.BZFlag.json]: Is `command` the executable that will be running when an icon is double clicked? If so, this should be the `bzflag` executable. `bzfs` is the server component of the game for hosting game servers.

Comment (rrigby) [reply org.bzflag.BZFlag.json]: I believe you are correct there.

**Comment by rrigby** [comment]
@allejo 
- Are there any specific resources we should familiarize ourselves with for becoming maintainers?

I personally just use other flatpak manifests as examples and get input from the flatpak/hub devs mostly. There are some docs you can follow however http://docs.flatpak.org/en/latest/first-build.html

- What does having write access entail?

You and anyone else who is a maintainer would be responsible for keeping the manifest up to date afaik.

File: org.bzflag.BZFlag.json (hunk 1 above)
Comment (rrigby) [reply org.bzflag.BZFlag.json]: I'm not quite sure what you mean by this suggestions.

**Comment by barthalion** [comment]
@allejo: There are also some notes on our wiki, mostly this article about maintenance: https://github.com/flathub/flathub/wiki/App-Maintenance

File: org.bzflag.BZFlag.json (hunk 1 above)
Comment (eszlari) [reply org.bzflag.BZFlag.json]: This is just a minor thing, but by using `-t` you don't have to repeat the (long) filename.

**Comment by allejo** [comment]
Thank you both for the links to the docs. I haven't personally tested this but this gets a 👍 from me for adding it to Flathub.

As for maintaining this, I'll take on the role as co-maintainer. Having write access would be nice. Just as a heads up, we'd likely be asking questions until we get the hang of things 😄

**Comment by barthalion** [comment]
That's fine, you can also ask anything on #flatpak channel on Freenode.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.bzflag.BZFlag

@allejo, make sure to accept the invite to get write access.

Thanks!

**Pull request closed without merging**